trantor/unittests/InetAddressUnittest.cc

```diff
@@ -11,6 +11,7 @@ TEST(InetAddress, innerIpTest)
     EXPECT_EQ(true, InetAddress("10.0.0.1", 0).isIntranetIp());
     EXPECT_EQ(true, InetAddress("172.31.10.1", 0).isIntranetIp());
     EXPECT_EQ(true, InetAddress("127.0.0.1", 0).isIntranetIp());
+    EXPECT_EQ(false, InetAddress("example.com", 0).isUnspecified());
```

Comment:
This line should be true

Reply:
And please add a positive test case.

Reply:
This should be false since InetAddress can't resolve FQDN to an IP address.